package.json

```diff
@@ -27,15 +27,23 @@
     "eslint": "^9.12.0",
     "@eslint/js": "^9.12.0"
   },
-  "bundleDependencies": [],
   "license": "BSD",
   "engines": {
     "node": ">=0.8.2"
```

Comment:
I think we should update minimum Node version. This one is "stone age"

Reply:
That's very true, but needs to be addressed in "master", not here